### DISCUSSION TITLE
Documentation on how to add more endpoints/APIs to this template

### DISCUSSION THREAD

**Discussion by codevalley** [fastapi/full-stack-fastapi-template#1421] [Questions]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched in the documentation/README.
- [X] I already searched in Google "How to do X" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs/README and didn't find an answer.

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```Text
from fastapi import APIRouter

from app.api.routes import items, login, users, utils, todo

api_router = APIRouter()
api_router.include_router(login.router, tags=["login"])
api_router.include_router(users.router, prefix="/users", tags=["users"])
api_router.include_router(utils.router, prefix="/utils", tags=["utils"])
api_router.include_router(items.router, prefix="/items", tags=["items"])
api_router.include_router(items.router, prefix="/todo", tags=["todo"])
```

### Description

I can see detailed documentation on forking the repo and also similar documentation on backend on deployment. 
Can you add some examples/documentation on how to add a new endpoint to this (once we fork this as a template). for example: If I am forking this to make a simple todo app. I will have to
- Add a new table(s) for the Todo list
- Add new crud endpoints
- Link with with the Auth
- Maybe make some changes to Front end to update the endpoints and data explorer. 

### Operating System

Windows, macOS

### Operating System Details

Mac/Windows (irrelevant for this question)

### Python Version

3.1.12

**Comment by polutica** [comment]
you need to create a new file under routes folder called todo.py

and initialize router by `router = APIRouter()` line

Your endpoints will be listed in docs page as you add them.

```
from app.models import (
    Message,
)

router = APIRouter()

@router.get("/someendpoint",, response_model=Message)
def put_some_message()->Any:
    return Message(message="here is your message")

```
